presto-docs/build

```diff
@@ -8,6 +8,6 @@ python3 -m venv presto-docs-venv
 source presto-docs-venv/bin/activate
 pip install -r requirements.txt
 
-make clean html
```

Comment:
Actually one quick thing--can we make this an optional flag?  So whoever is building the docs doesn't need to additionally install the LaTeX dependencies

Reply:
I would love it if the PDF build was optional with an added flag. The PDF build takes longer than the Sphinx RST->HTML build, so the combination more than doubles my compile time for local doc builds, which would affect me as I often do that a dozen+ or more times daily.

Reply:
Done, there's a new flag to build PDF optionally, `--with-pdf`. I've updated the README


Re: long time to build PDF version, I think there might be something odd on your side, Steve. As it's pretty fast on CI and on my machine as well :D

Reply:
> Done, there's a new flag to build PDF optionally, `--with-pdf`. I've updated the README

Pulled updated branch and the flag works great: `./build` builds only HTML, and `./build --with-pdf` builds HTML and PDF. 

Importantly, `./build --with-pdf` then `./build` deletes the target/latex PDF output directory, as expected. 

Nice work! 

> Re: long time to build PDF version, I think there might be something odd on your side, Steve. As it's pretty fast on CI and on my machine as well :D

`./build` takes 16 seconds to build the HTML docs on my machine. 
`./build --with-pdf` takes 59 seconds to build the HTML and PDF on my machine. That is pretty fast but it is 369% as long a time as just the HTML :D.